### PR TITLE
CIRC-1500 Document new-cluster reconstitute method

### PR DESCRIPTION
**Pull request by esproul**

### CODE REVIEW

File: migrating-clusters.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+http://<node ip address>:<node port>/stats.json
+```
+within the "reconstitute" section. Note that there may not be messages
+appearing on the console while this runs. This is normal; do not stop the
+reconstitute. Current progress will be saved - if the process stops for any
+reason, everything should pick back up approximately where it was. If the
+download stops partway for any reason, you may resume it with the following
+command:
+```
+/opt/circonus/bin/irondb-start -B -T <topo_hash> -O
```
Comment (pamaddox) [inline migrating-clusters.md]: Maybe <source cluster topo_hash> here?

Comment (pamaddox) [inline migrating-clusters.md]: Maybe `<source cluster topo_hash>` here?

File: migrating-clusters.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+where the argument to `-O` is the IP address and port of a node in the source
+cluster. The port is the cluster API port, typically 8112. The reconstitute
+will get the topology information from the source cluster node using the specified
+topology. Actual metric data fetches will be done against all source cluster
+nodes, using the topology information to determine the primary owner of each
+metric stream.
+1. Wait until the reconstitute operation has fetched 100% of its data from
+the source cluster. You can access the current percentage done at:
+```
+http://<node ip address>:<node port>/stats.json
```
Comment (pamaddox) [inline migrating-clusters.md]: You can just go to `http://<node ip address>:<node port>` and there will be a dynamic thing that shows current progress that updates in real time - don't need to go to `stats.json`

File: migrating-clusters.md (hunk 1 above)
Comment (esproul) [reply migrating-clusters.md]: Sure, can change that.

File: migrating-clusters.md (hunk 2 above)
Comment (esproul) [reply migrating-clusters.md]: That's right, there's a UI for this now. :)

**Pull request merged**